lib/navigation/preprocess_encoders.cpp

```diff
@@ -1,91 +1,94 @@
+#include "consts.hpp"
 #include "preprocess_encoders.hpp"
 
+#include "core/types.hpp"
+
 namespace hyped::navigation {
 
 EncodersPreprocessor::EncodersPreprocessor(core::ILogger &logger)
     : logger_(logger),
-      encoder_outliers_({0, 0, 0, 0}),
-      encoders_reliable_({true, true, true, true}),
-      num_reliable_encoders_(core::kNumImus)
+      num_consecutive_outliers_per_encoder_({0, 0, 0, 0}),
+      is_reliable_per_encoder_({true, true, true, true}),
+      num_reliable_encoders_(core::kNumImus),
+      max_consecutive_outliers_(10)
```

Comment:
What's the benefit of initializing here as opposed to just storing it as a constant class member?

Reply:
this is a tricky question, the short answer is consistency

if we were to agree that, within HYPED, we use in-class initialisation wherever possible then i wouldn't have anything to object to; however, since lots of these were already being initialised here, it feels weird to do it in two places